atom/common/api/resources/ipc_bindings.js

```diff
@@ -5,19 +5,24 @@ exports.didCreateDocumentElement = function() {
   if (!window)
     return
 
-  var slice = [].slice;
+  var slice = [].slice
 
   window.alert = function(message, title) {
-    return ipc.send('window-alert', message, title);
+    return ipc.send('window-alert', message, title)
   }
 
   window.confirm = function(message, title) {
-    return ipc.sendSync('window-confirm', message, title);
+    return ipc.sendSync('window-confirm', message, title)
   }
 
   window.prompt = function(text, defaultText) {
-    return ipc.sendSync('window-prompt', text, defaultText);
+    return ipc.sendSync('window-prompt', text, defaultText)
   }
+
+  if (window.chrome && window.chrome.tabs && window.chrome.tabs.create)
+    window.open = function(url, name, specs, replace) {
+      return window.chrome.tabs.create({url: url})
```

Comment:
not handling the other args is ok for now, but we should at least throw an error if anything other than url is passed